turbinia/e2e/setup-gce.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+# This script will
+# - setup a GCE Ubuntu 18.04 LTS base image for Turbinia e2e testing.
+# - execute Terraform to setup the Turbinia environment.
+# Make sure the service account your GCE instance is running under has full API scope
+# access and is project owner for Terraform to function correctly.
+
+install_packages() {
+  apt-get -y update
+  apt-get -y install python-pip python-virtualenv unzip
+}
+
+echo "Initiate gcloud configuration"
+
+if [ $# -ne  1 ]
+then
+  echo "Not enough arguments supplied, please provide project."
+  echo "$0 [PROJECT]"
+  exit 1
+fi
+
+PROJECT="$1"
+
+gcloud --project=$PROJECT info
+
+# Try to install the packages. GCE instances do their own apt-get updating at
+# first boot so we have to wait on that to install our packages.
+max_retry=100
+for try in $(seq 1 ${max_retry}); do
+  [[ ${try} -gt 1 ]] && sleep 10
+  install_packages && exit_code=0 && break || exit_code=$?
+  echo "Failed to apt-get install needed packages, retrying in 10 seconds."
+done;
+
+# Install Terraform
+wget -q -O terraform.zip https://releases.hashicorp.com/terraform/0.12.29/terraform_0.12.29_linux_amd64.zip
+unzip terraform.zip && cp terraform /usr/local/bin/ && rm terraform.zip terraform
+
+# Git clone Turbinia Terraform scripts
+git clone https://github.com/forseti-security/osdfir-infrastructure.git
+
+# Deploy Turbinia infrastructure with terraform
+# If you see "ERROR: (gcloud.app.create) PERMISSION_DENIED: The caller does not have permission"
+# Only GCP Project Owner can create App Engine project
+# See documentation: https://cloud.google.com/appengine/docs/standard/python/console/#create
+echo "Enable AppEngine API and sleep to make sure service is enabled."
+gcloud --project=$PROJECT services enable appengine
```

Comment:
This can be run on new or existing cloud projects. I had thought about doing the below but removed it as trying to enable to appengine when it already is enabled just throws an error saying that which does not harm. I thought that comparing strings and using filters as below would be more error prone...wdyt?

A=gcloud services list --enabled --filter='name:appengine.googleapis.com'`
if [ "$A" = "Listed 0 items." ]
then
    echo "Enabling App Engine"
fi

Reply:
Got it. Yeah was wondering if the error would stop the rest of the flow and since it doesn't then not as big of an issue and can see why you left it out.

If you did want to make it more error prone, then the string comparison above LGTM!